pkgs/development/python-modules/etebase/default.nix

```diff
@@ -0,0 +1,62 @@
+{ lib
+, wheel
+, rustPlatform
+, pipInstallHook
+, setuptools-rust
+, python
+, msgpack
+, requests
+, openssl
+, perl
+, rustfmt
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "etebase";
+  version = "0.31.1";
+
+  src = fetchFromGitHub {
+    owner = "etesync";
+    repo = "etebase-py";
+    rev = "950902a520417eaaf34d2c5e5e590ddb3c90358e";
+    sha256 = "163iw64l8lwawf84qswcjsq9p8qddv9ysjrr3dzqpqxb2yb0sy39";
+    fetchSubmodules = true;
+  };
+
+
+  cargoSha256 = "0w8ypl6kj1mf6ahbdiwbd4jw6ldxdaig47zwk91jjsww5lbyx4lf";
+
+  nativeBuildInputs = [
+    rustfmt
```

Comment:
Why do we need to format the code?

Reply:
It's a dependency of flapigen and therefore also required for etebase.